### DISCUSSION TITLE
Log calls with string parameter

### DISCUSSION THREAD

**Discussion by samhill303** [touchlab/Kermit#71] [Ideas]
We currently only accept lambda's for the message parameter. This ensures we dont evaluate the string if no logger cares about it. This is good but a little clunky if you dont care about that. Consider adding a string only version

**Comment by samhill303** [comment]
This didn't feel too clunky when integrating in KaMPKit. Particularly in android studio. A little worse in iOS. This might be worth doing, but not highest priority IMO

**Comment by juanchosaravia** [comment]
I feel it in the same way, it's kind of odd passing a lambda for just sending a String. Seems like too much overhead.
